### DISCUSSION TITLE
Supported on RHEL 7.9?

### DISCUSSION THREAD

**Discussion by jxu** [mkleehammer/pyodbc#1242] [Q&A]
### Environment

To diagnose, we usually need to know the following, including version numbers.  On Windows, be
sure to specify 32-bit Python or 64-bit:

- Python: 3.6.8
- pyodbc: n/a
- OS: RHEL 7.9
- DB: SQL Server
- driver: UnixODBC

### Issue

Is it possible to install pyodbc on RHEL 7.9? It seems to require python3-devel which is maybe not supported, because I get "No package python3-devel available"
 https://stackoverflow.com/questions/43047284/how-to-install-python3-devel-on-red-hat-7

**Comment by v-makouz** [comment]
You might be able to install python3-devel manually if you download the rpm, but it might have dependencies that are also missing. (Assuming none of the suggestions from the Stackoverflow worked)

**Comment by jxu** [comment]
I wasn't able to download python3-* packages from EPEL or RHSCL due to subscriptions being messed up. But we have since moved away from this old server to Azure ML which seems to have pyodbc out of the box.
